### PR TITLE
Add Trailhead email templates

### LINKED ISSUE #1073: Support email variations for Trailhead signup
Trailhead will be using a custom email template for new users. From `about:welcome` page, they will propagate a param `style=trailhead`. We should pass this param to the auth-server `/account/create` and let it use the new template. All other signups will use the previous email templates.

New template: https://docs.google.com/document/d/1I7TSNa7mvUFipX48OhGSRbuVybIqSb5QdnCmivSjl-c/edit#

Latest designs: https://mozilla.invisionapp.com/share/3GRZJ9J65A9#/363231220_Email_-_Verified

### PR DESCRIPTION
**Pull request by vbudhram**
This is a draft PR for adding Trailhead email templates.

- [x] Confirm account template
- [x] Account confirmed template
- [x] auth-server to handle `style=trailhead`
- [x] Supporting tests
- [x] Replace placeholder images

Fixes #1073

### CODE REVIEW

**Comment by shane-tomlinson** [comment]
Will we need the new logo used in @ryanfeeley's new [Invision designs](https://mozilla.invisionapp.com/share/3GRZJ9J65A9#/screens/362776357/comments)?

Screenshot 2019-05-13 at 21 46 50

**Comment by vbudhram** [comment]
@shane-tomlinson Yea we will, I'm currently using  https://imgur.com/a/2CMat0g, until we get the image and host it like https://github.com/mozilla/fxa/blob/2412e3c36a269b5f83ee0d950fc8f761c3cb5288/packages/fxa-auth-server/lib/senders/partials/base/base.html#L21

**Comment by vbudhram** [comment]
Updated email templates to match designs. I'm using place holder images in the meantime.

Confirm account:
Screen Shot 2019-05-14 at 1 58 11 PM

Account verified:
Screen Shot 2019-05-14 at 1 57 57 PM

**Comment by shane-tomlinson** [comment]
> Updated email templates to match designs. I'm using place holder images in the meantime.

I'll send you the images bundle in Slack.

**Comment by vbudhram** [comment]
@mozilla/fxa-devs This PR is waiting for images but otherwise is in good shape to be reviewed.

**Comment by shane-tomlinson** [comment]
One thing brought up with @ryanfeeley is that the new logo should be on all emails.

**Comment by vbudhram** [comment]
> One thing brought up with @ryanfeeley is that the new logo should be on all emails.

Sounds good, that might make things a little easier 👍🏽

File: packages/fxa-auth-server/lib/senders/email.js [hunk 1]
```diff
@@ -996,16 +1007,25 @@ module.exports = function (log, config, oauthdb) {
...
 
-    const templateName = 'postVerifyEmail';
-    const links = this._generateLinks(this.syncUrl, message.email, {}, templateName);
+    let templateName = 'postVerifyEmail';
+    let subject = gettext('Firefox Account verified');
+    const query = {};
+
+    if (message.style === 'trailhead') {
+      templateName = 'postVerifyTrailheadEmail';
+      subject = gettext('Welcome to the Firefox family');
```
Comment (shane-tomlinson) [inline packages/fxa-auth-server/lib/senders/email.js]: This string was changed to "Your Firefox Account is Confirmed" - see [the updated strings doc](https://docs.google.com/document/d/1jQwIsAMz7gFDUa0f22LVoAhKxo5vr2I3_xf8Kib2MVs/edit?ts=5ce20322#bookmark=id.991f3pix04kh).

**Comment by vbudhram** [comment]
@shane-tomlinson Updated!

**Pull request merged**